none1.html

```diff
@@ -0,0 +1,73 @@
+<html>
+<head>
+    <link rel="stylesheet" type="text/css" href="bootstrap.min.css"> </link>
+    <style type="text/css" media='screen,projection'>
+
+    @import = url("https://cdn.datatables.net/1.10.16/css/jquery.dataTables.min.css");
+    @import = url("https://cdn.datatables.net/buttons/1.4.2/css/buttons.dataTables.min.css");
+    @import= url("https://cdn.datatables.net/select/1.2.3/css/select.dataTables.min.css");
+    @import= url("https://editor.datatables.net/extensions/Editor/css/editor.dataTables.min.css");
+</style>
+</head>
+    <body>
+        <p class="h2 text-primary text-center">List of invitees</p>
+                <table class="table">
+                    <thead class='thead-dark'>
+<table id="table" class="display" cellspacing="0" width="50%">
+        <thead >
+            <tr>
+                <th>sn</th>
+                <th>first_name</th>
+                <th>last_name</th>
+                <th>coinid</th>
+                <th>Status</th>
+            </tr>
+        </thead>
+        <script type="text/javascript">
+        var editor;
+
+        (document).ready(function() {
```

Comment:
`$(document).ready(...` is not Javascript - it's from a javascript library called jQuery. To use jQuery, you should first load the jQuery javascript library. Put 
```
<script src="http://code.jquery.com/jquery-latest.min.js" type="text/javascript"></script>
```
inside `<head></head>` or before `<script></script>` tag in your code, to load the jquery library.

Reply:
Also, if you're using jQuery, it's `$(document).ready`, not: `(document).ready`.